mmdet/models/detectors/soft_teacher.py

```diff
@@ -69,20 +69,17 @@ def loss_by_pseudo_instances(self,
 
         x = self.student.extract_feat(batch_inputs)
 
-        losses = {}
+        pseudo_losses = {}
```

Comment:
we do not need to rename it as pseudo_losses, we can simply use losses and return the renamed ones